airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/config.py

```diff
@@ -166,6 +166,7 @@ class ClientContainerConfig(BaseConfig):
     setup_command: List[str] = Field(None, description="Command for running the setup/teardown container for setup.")
     teardown_command: List[str] = Field(None, description="Command for running the setup/teardown container for teardown.")
```

Comment:
now we have final_teardown_command, can we mark teardown_command as optional?

Reply:
Yep! I'll push an update for that.

Reply:
Done.